packages/syft/examples/experimental/trask/Untitled3.ipynb

```diff
@@ -0,0 +1,279 @@
+       "sqlalchemy.orm.decl_api.DeclarativeMeta"
+      ]
+     },
+     "execution_count": 29,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "type(Base)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "id": "ce102810-affd-427d-9c98-057f7654c148",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class User(Base):\n",
+    "    __tablename__ = 'users'\n",
+    "    id = Column(Integer, primary_key=True)\n",
+    "    name = Column(String)\n",
+    "    fullname = Column(String)\n",
+    "    nickname = Column(String)\n",
+    "\n",
+    "    def __repr__(self):\n",
+    "        return \"<User(name='%s', fullname='%s', nickname='%s')>\" % (\n",
+    "                            self.name, self.fullname, self.nickname)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "id": "85500fb6-82e8-4755-9ce6-a5eea07b42a5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Base.metadata.create_all(engine)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "id": "807706b6-bd66-4ea8-95a6-d98092c7e07c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ed_user = User(name='ed', fullname='Ed Jones', nickname='edsnickname')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "id": "9d2c041a-14f4-4fc8-905d-6ed6e2d2c20e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sqlalchemy.orm import sessionmaker\n",
+    "Session = sessionmaker(bind=engine)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "id": "7614969d-ba6a-4e50-8aec-788c29806c7d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "session = Session()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "id": "3717b22b-a7bf-43fb-aefe-a744f358ed85",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "sqlalchemy.orm.session.Session"
+      ]
+     },
+     "execution_count": 36,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "type(session)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "5e2e85c4-e455-47dd-bbad-cdf808cb8b12",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "session.add(ed_user)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c67c147d-a659-40fa-9b35-148a15e8cf31",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.10"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
Before merge, we should stick only to the relevant notebooks (like create a story of 3 notebooks that make some functionality). Currently, there are way too many notebooks.

Reply:
I think "experimental" can be messy though.